up42/order.py

```diff
@@ -46,9 +44,9 @@ def info(self) -> dict:
         """
         Gets and updates the order information.
         """
-        url = f"{self.auth._endpoint()}/workspaces/{self.workspace_id}/orders/{self.order_id}"
```

Comment:
self.workspace_id is not used, let's drop it as well

Reply:
This is still used in the estimate and place endpoints, but they will be removed in the following migration tickets.

Reply:
It's not used there as well, auth.workspace_id is used instead